### PR TITLE
🌐 Add Chinese translation for Advanced - Return a Response Directly

### PR DESCRIPTION
**Pull request by RunningIkkyu**
Add Chinese docs for advanced/response-directly

### CODE REVIEW

File: docs/zh/docs/advanced/response-directly.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# 直接返回响应
+
+当你创建一个 **FastAPI** *路径操作* 时，你可以正常返回以下任意一种数据：`dict`，`list`，Pydantic 模型，数据库模型等等。
+
+**FastAPI** 默认会使用 `jsonable_encoder` 将这些类型的返回值转换成 JSON 格式，`jsonable_encoder` 在 [JSON 编码器兼容](../tutorial/encoder.md){.internal-link target=_blank} 中有阐述。
```
Comment (waynerv) [inline docs/zh/docs/advanced/response-directly.md:5]: "JSON Compatible Encoder" should be "JSON兼容编码器“

File: docs/zh/docs/advanced/response-directly.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+直接返回响应可能会有用处，比如返回自定义的响应头和 cookies。
+
+## 返回 `Response` 
+
+事实上，你可以返回任意 `Response` 或者任意 `Response` 的子类。
+
+!!! tip "小贴士"
+    `JSONResponse` 本身是一个 `Response` 的子类。
+
+当你返回一个 `Response` 时，**FastAPI** 会自动忽略它。
```
Comment (waynerv) [inline docs/zh/docs/advanced/response-directly.md:20]: ”自动忽略“ could be ”直接传递“

File: docs/zh/docs/advanced/response-directly.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+!!! note "技术细节"
+    你也可以使用 `from starlette.responses import JSONResponse`。
+
+    出于方便，**FastAPI** 会提供与 `starlette.responses` 相同的 `fastapi.responses` 给开发者。但是大多数可用的响应都直接来自 Starlette。
+
+## 返回自定义 `Response`
+
+The example above shows all the parts you need, but it's not very useful yet, as you could have just returned the `item` directly, and **FastAPI** would put it in a `JSONResponse` for you, converting it to a `dict`, etc. All that by default.
```
Comment (waynerv) [inline docs/zh/docs/advanced/response-directly.md:46]: This line should be deleted.

File: docs/zh/docs/advanced/response-directly.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+你可以把你的 XML 内容放到一个字符串中，放到一个 `Response` 中，然后返回。
+
+```Python hl_lines="1  18"
+{!../../../docs_src/response_directly/tutorial002.py!}
+```
+
+## 说明
+
+当你直接返回 `Response` 时，它的数据既没有校验，又不会进行转换（序列化），也不会进行自动化文档。
```
Comment (waynerv) [inline docs/zh/docs/advanced/response-directly.md:62]: "进行自动化文档" is a little weird.May be we should use "自动生成文档” here.

**Review by waynerv — CHANGES_REQUESTED** [review]

**Comment by RunningIkkyu** [comment]
@Dustyposa @phy25 @yanever @Laineyzhang55  @waynerv Please take a look.

**Review by waynerv — APPROVED** [review]

**Review by Laineyzhang55 — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome! Thank you @RunningIkkyu :cake: 

And thanks @waynerv and @Laineyzhang55 for the reviews :mag: :coffee:

**Pull request merged**